packages/fhir-care-team/src/components/CreateEditCareTeam/tests/form.unmocked.test.tsx

```diff
@@ -193,9 +199,101 @@ test('filter select by text able to create new careteam', async () => {
   wrapper.find('form').simulate('submit');
 
   await waitFor(() => {
-    expect(successNoticeMock.mock.calls).toEqual([['Successfully Added Care Teams']]);
+    expect(successNoticeMock.mock.calls).toEqual([['Successfully added CareTeams']]);
   });
 
   expect(nock.isDone()).toBeTruthy();
   wrapper.unmount();
 });
+
+test('1157 - Create care team works corectly', async () => {
+  const successNoticeMock = jest
+    .spyOn(notifications, 'sendSuccessNotification')
+    .mockImplementation(() => undefined);
+
+  nock(props.fhirBaseURL)
+    .put(`/${careTeamResourceType}/${createdCareTeam2.id}`, createdCareTeam2)
+    .reply(200)
+    .persist();
+
+  render(<CareTeamForm {...props} />);
+
+  await waitFor(() => {
+    expect(screen.getByText(/Create Care Team/)).toBeInTheDocument();
```

Comment:
nitpick: I prefer destructuring the query methods from render instead of screen. e.g `const { getByText } = render(<CareTeamForm {...props} />);`

Reply:
https://kentcdodds.com/blog/common-mistakes-with-react-testing-library#not-using-screen